packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNavSelectedIndicator.js

```diff
@@ -27,15 +27,15 @@ import {
   type Theme,
 } from 'react-native-bpk-theming';
 import {
-  colorBlue700,
+  colorSkyBlue,
   borderSizeLg,
 } from 'bpk-tokens/tokens/base.react.native';
 
 import { REQUIRED_THEME_ATTRIBUTES, themePropType } from './theming';
 
 const styles = StyleSheet.create({
   selectedIndicator: {
-    backgroundColor: colorBlue700,
+    backgroundColor: colorSkyBlue,
```

Comment:
We should check if this blue should be Sky Blue and not another Blue

Reply:
This is what we have for Android currently. I'll check with Will